docs/examples/blog/index.html

```diff
@@ -169,6 +169,7 @@ <h2 class="blog-post-title">New feature</h2>
     ================================================== -->
     <!-- Placed at the end of the document so the pages load faster -->
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+    <script>window.jQuery || document.write('<script src="../../assets/js/vendor/jquery-1.11.3.min.js"><\/script>')</script>
```

Comment:
I thought about it, but I went with this. I don't have any strong opinion since GitHub doesn't really cache the CSS/JS files for a long time, so it should be OK if we use the same filename.